bundles/org.openhab.binding.linky/src/main/resources/OH-INF/thing/group-linky-local-historical-ejp.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="linky"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="linky-local-historical-ejp">
+		<label>Historical EJP</label>
+		<channels>
+			<channel id="ejphn" typeId="consumption-type">
+				<label>EJP Index Standard Hours</label>
+				<description>Index EJP option Standard Hours</description>
+			</channel>
+
+			<channel id="ejphpm" typeId="consumption-type">
+				<label>EJP Index High Hours</label>
+				<description>Index EJP option High hours</description>
+			</channel>
```

Comment:
To be checked because I have the strange feeling (after getting warnings in logs that type "consumption-type" does not exist!

Reply:
It has been fixed now.